### ISSUE TITLE
Project search results : Double Click to Open results , or Confirmation if file is too huge..

### ISSUE THREAD

**Issue by v3ss0n** [atom/find-and-replace#389]
When searching project files , it usually lead to include huge files , When accidentally clicked  it causes atom to crash.

Huge file /Long file support is long way to go so would be nice if:

- confirmation if the file with result is too huge
- Or , Double click to open search results when project search.

**Comment by bronson** [comment]
This is true.  It's probably the most common way to crash Atom now.  At least I know it is for me.

**Comment by v3ss0n** [comment]
Does that still happens for you? Long File/ Huge file support is already landed and since then it is fixed for me.

**Comment by bronson** [comment]
Oh, don't get me wrong, it's TONS better than 2014.  This is probably not a big deal, but accidentally opening minimized js files and giant logfiles can cause some trouble.

**Comment by v3ss0n** [comment]
opening is fine now .
But what crashes now is searching inside big files , eveng 1mb minified file is crashing.
